npc/re/instances/GhostPalace.txt

```diff
@@ -905,9 +905,9 @@ OnTimer28500:
 	mes "Please help me save the soul of Sakray.";
 	next;
 	setarray .@item_name$[0],
-		"Thanatos Sword", "Thanatos Great Sword", "Thanatos Spear", "Thanatos Long Spear", "Thanatos Staff",
-		"Thanatos Two-handed Staff", "Thanatos Bow", "Thanatos Dagger", "Thanatos Katar", "Thanatos Knuckle",
-		"Thanatos Hammer", "Thanatos Axe", "Thanatos Violin", "Thanatos Whip",
+		"Thanos Sword", "Thanos Broadsword", "Thanos Spear", "Thanos Two-Handed Spear", "Thanos Staff",
```

Comment:
I would rather change the item name in item_db and call the name from there.

Reply:
Yah, I'm not good at scripting, I'll leave that part for you guys.